.lintr

```diff
@@ -1,6 +1,10 @@
 linters: linters_with_defaults(
    line_length_linter(120),
```

Comment:
This line_length_linter can be disabled or changed to a different character length if needed. 

Implicit_integer_linter makes sure 1L is used for integers instead of 1. It's not part of the defaults in zzz.R, so it can just be removed. But it was set by r-lib in their config.